### DISCUSSION TITLE
I it possible to programatically close AlertDialog?

### DISCUSSION THREAD

**Discussion by yinkakun** [radix-ui/primitives#1562] [Help]
When I try closing it by returning null when the check passes, it still leaves a `pointer event: none` style on the body, making it impossible to click anything else on the page.

**Comment by jjenzz** [comment]
you can use the `open` and `onOpenChange` props to make the component _controlled_ and then you can `setOpen(false)` to close it:

```jsx
const [open, setOpen] = React.useState(false);

return (
  <AlertDialog.Root open={open} onOpenChange={setOpen}>
   /* ... */
  </AlertDialog.Root>
)
```
